packages/console/src/pages/ConnectorDetails/components/SenderTester/index.tsx

```diff
@@ -56,21 +66,30 @@ const SenderTester = ({ connectorId, connectorType, config, className }: Props)
     };
   }, [showTooltip]);
 
-  const onSubmit = handleSubmit(async (formData) => {
-    const { sendTo } = formData;
+  const configuration = useMemo(() => {
+    if (connectorFormItems) {
```

Comment:
why do we need explicitly pass this while the component is in a form context?

Reply:
senderTester and connector config form are sibling form components, passwordless message test uses config in the current connector config form.